File: src/test/java/kitchenpos/product/ProductFixture.java

package kitchenpos.product;

import kitchenpos.common.vo.Name;
import kitchenpos.common.vo.Price;
import kitchenpos.product.domain.Product;

import java.math.BigDecimal;
import java.util.UUID;

public class ProductFixture {

    public static Product product(UUID id, BigDecimal productPrice) {
        return new Product(id, new Name("상푸명", false), new Price(productPrice));
    }
}

File: src/test/kotlin/kitchenpos/product/ProductFixture.kt
package kitchenpos.product

import kitchenpos.domain.Product
import java.math.BigDecimal


class ProductFixture {

    companion object {
        private const val DEFAULT_NAME: String = "테스트 상품"
        private val DEFAULT_PRICE: BigDecimal = BigDecimal.valueOf(1000L)

        fun fixture(
            name: String? = DEFAULT_NAME,
            price: BigDecimal? = DEFAULT_PRICE
        ): Product {
            val product = Product()
            product.name = name
            product.price = price
            return product
        }
    }
}

